cassandane/Cassandane/Cyrus/Prometheus.pm

```diff
@@ -263,7 +263,6 @@ sub test_quota_commitments_no_improved_mboxlist_sort
 {
     my ($self) = @_;
 
-    $self->{instance}->{config}->set('improved_mboxlist_sort', 'no');
```

Comment:
I think you can remove the whole `sub test_quota_commitments_no_improved_mboxlist_sort`, since there's no special behaviour left for it to test.